crates/m3/src/builder/table.rs

```diff
@@ -320,6 +324,8 @@ pub struct Table<F: TowerField = B128> {
 	pub id: TableId,
 	pub name: String,
 	pub columns: Vec<ColumnInfo<F>>,
+	/// Whether the table size is required to be a power of two.
+	pub po2_sized: bool,
```

Comment:
Maybe `requires_power_of_two_size` would be a more descriptive name here

Reply:
I changed to `power_of_two_sized`